_posts/2015-06-03-team-offsite.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "Team ePages 6 offsite event"
+date: "2015-06-03 13:44:00"
+icon: users
+tags: team event offsite
+categories: events
+authors: ["Fouad-Steffen Nabhan"]
+---
+
+Working hard and embrace the nature? How does that fit together? Our ePages 6 development team organised a team offsite event and convened in a lovely castle to focus on knowledge transfer. What did they do? Read on!
```

Comment:
"our ePages 6 development team organised an offsite event" no need to write team twice.